test/helpers/rest.py

```diff
@@ -32,7 +32,7 @@
 from string import ascii_uppercase as str_alpha
 from string import digits as str_num
 
-from apprise import plugins
```

Comment:
> Part of the reason i imported the plugins namespace everywhere was because [...]

I see.

> [Removing it], I don't think it will cause an issue though [...]

Excellent. I think it was the root cause of the problems we encountered here. Great to hear if you think it will work without this trick.